### DISCUSSION TITLE
How can I implement cookies wrapper?

### DISCUSSION THREAD

**Discussion by swassapparel1** [nodejs/undici#1572] [Q&A]
Hi everyone, just wanted to ask if it's possible to wrap a cookie wrapper in unidici? Because currently I'm using node-fetch and using fetch-cookie wrapper. I want to switch to unidici fetch and wanted to try if it will increase the performance compared to node-fetch. Many thanks!

**Comment by mcollina** [comment]
I don't think this is easy to do at the moment as we are debating at wintercg what's the best way to have spec-compatible cookie support. 

You can read more at: https://github.com/wintercg/fetch/issues/6
